### ISSUE TITLE
Issue highlighting & formatting text

### ISSUE THREAD

**Issue by leemediagroup** [WordPress/gutenberg#5105]
## Issue Overview
This is a note from one of our clients that is testing the newest beta version of WordPress and the beta version of Gutenberg. They have high-traffic sites with a lot of content, so they want to be sure they fully understand the new features before the final update is released, so we've had them testing it out. This is one of the concerns they have come across so far: 

_I write a ton of content on my site, such as deals and special offers. Gutenberg does NOT allow me to highlight or choose a single word or phrase (like the word FREE) that can appear in a different color (such as red when the rest of the text is black) and also I can't highlight a phrase in yellow, for example._

Basically, when they try to make edits to content like bolding or changing the color of a word, it is change the entire sentence.

**Comment by mtias** [comment]
Could you provide us with more details about this? Formatting buttons work as expected for me.

**Comment by karmatosed** [comment]
This is working as expected for me also, for now closing this. If you do get further issues @leemediagroup please let us know and we can always reopen.

**Comment by phpbits** [comment]
@leemediagroup I've added text highlighting color options on EditorsKit plugin ( https://wordpress.org/plugins/block-options/ ) and I hope this solves your problem. Here's how my implementation works :

![EditorsKit-Text-Colors](https://user-images.githubusercontent.com/3365507/57937370-60a29100-78f8-11e9-8069-06a8ec43259d.gif)
